exercises/space-age/description.md

```diff
@@ -10,8 +10,9 @@ Given an age in seconds, calculate how old someone would be on:
    - Neptune: orbital period 164.79132 Earth years
 
 So if you were told someone were 1,000,000,000 seconds old, you should
-be able to say that they're 31.69 Earth-years old. Round all ages to
-the nearest hundredth of a year.
+be able to say that they're 31.69 Earth-years old. Because floating-point
+math is not exact, your answer only needs to be correct to two decimal places
+(but you are free to leave your answer unrounded).
```

Comment:
My impression of this is that the rounding will be handled by the test suite either by actual rounding or through tolerancing.  This puts a potential burden on the maintainers to process the students result correctly.....  I think the burden would be minimal.  On the other hand.... see my comment in #946 .

Reply:
> the rounding will be handled by the test suite either by actual rounding or through tolerancing

yes

> This puts a potential burden on the maintainers to process the students result correctly

yes, 

Note that the burden is present regardless of whether this sentence is present. If it needs to happen anyway, might as well inform students of the policy.

Only problem is if a track desires to use a different policy. So, should problem-specifications define a default policy, at which point differing tracks will modify the README as necessary? Or, should problem-specifications define no policy, in which case every track that wishes to define any policy (as would be helpful to students) must do so in `.meta/hints.md` or via some other method?

Reply:
`(but you are free to leave your answer unrounded)` this statement may not be necessary.  I think this may be why I am liking #946 more.  This statement makes the description feel a little loose.

Reply:
I can thus propose "This means you are free to leave your answer unrounded"

Reply:
(that assumes the "But" was the only loose part about it. If instead it was the "you are free", then it would have to be "This means you should leave your answer unrounded")

Reply:
(that assumes the "But" was the only loose part about it. If instead it was the "you are free", then it would have to be "This means you should leave your answer unrounded")